react-components/stories/HighlightNode.stories.tsx

```diff
@@ -0,0 +1,90 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import {
+  CadModelContainer,
+  type FdmAssetMappingsConfig,
+  RevealContainer,
+  RevealToolbar,
+  useReveal
+} from '../src';
+import { CogniteClient } from '@cognite/sdk';
+import { Color, Matrix4 } from 'three';
+import { type ReactElement, useEffect, useState } from 'react';
+import { type PointerEventData } from '@cognite/reveal';
+import { queryMappedData } from '../src/components/Reveal3DResources/queryMappedData';
+import { useFdmSdk, useSDK } from '../src/components/RevealContainer/SDKProvider';
+
+const DefaultFdmConfig: FdmAssetMappingsConfig = {
+  source: {
+    space: 'hf_3d_schema',
+    version: '1',
+    type: 'view',
+    externalId: 'cdf_3d_connection_data'
+  },
+  global3dSpace: 'hf_3d_global_data',
+  assetFdmSpace: 'hf_customer_a'
+};
+
+const meta = {
+  title: 'Example/HighlightNode',
+  component: CadModelContainer,
+  tags: ['autodocs']
+} satisfies Meta<typeof CadModelContainer>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const token = new URLSearchParams(window.location.search).get('token') ?? '';
+const sdk = new CogniteClient({
+  appId: 'reveal.example',
+  baseUrl: 'https://greenfield.cognitedata.com',
+  project: '3d-test',
+  getToken: async () => await Promise.resolve(token)
+});
+
```

Comment:
There is a helper function `createSdkByUrlToken` that does exactly that, that is used across most stories (not sure why it was not used in toolbar and some others).